vector/src/main/res/layout/fragment_room_member_list.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
A FrameLayout is enough to do this, with `android:layout_gravity="bottom|end"` for the FAB

Reply:
Cool.